### DISCUSSION TITLE
5月11和5月12连着2天封端口

### DISCUSSION THREAD

**Discussion by zsanjin-p** [v2fly/v2ray-core#2506] [General]
用的是nginx+ws+tls，连着两天一觉起来端口封了，用cloudflare代理ip，但是cf的tls设置是灵活，没有完全加密，现在都是啥情况，我还有开心魔法的梯子，5月11也挂了

**Comment by likooo12580** [comment]
墙变聪明了呗

**Comment by qianbinbin** [comment]
我最近疑似遭遇了严重的 SNI 阻断，域名本身的网站可以正常访问，但是油管、GitHub 等网站断断续续
方案用过 VLESS+WS+TLS 和 VMess+WS+TLS，套不套 CDN 都一样
